examples/with-sentry/package.json

```
@@ -6,8 +6,8 @@
     "start": "next start"
   },
   "dependencies": {
-    "@sentry/nextjs": "^6.3.5",
-    "next": "latest",
+    "@sentry/nextjs": "^6.13.4",
```

Just updating to the newest version.

```suggestion
    "@sentry/nextjs": "^6.14.1",
```